lib/inputs/node/static.ts

```diff
@@ -2,12 +2,25 @@ import { basename } from "path";
 import { ExtractAction } from "../../extractor/types";
 import { streamToString } from "../../stream-utils";
 
-const nodeAppFiles = ["package.json", "package-lock.json", "yarn.lock"];
-const deletedAppFiles = nodeAppFiles.map((file) => ".wh." + file);
+const nodeAppFilePatterns = [
+  /^package\.json$/,
```

Comment:
Why not check explicitly if the files are package.json / package-lock.json / etc.? Why do we need the regex?